packages/rum_sdk/example/lib/main.dart

```diff
@@ -4,25 +4,27 @@ import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:offline_transport/offline_transport.dart';
 import 'package:rum_sdk/rum_sdk.dart';
-import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:http/http.dart' as http;
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   HttpOverrides.global = RumHttpOverrides(HttpOverrides.current);
-  await dotenv.load(fileName: ".env");
-  RumFlutter()
-      .transports
-      .add(OfflineTransport(maxCacheDuration: const Duration(days: 3), collectorUrl: dotenv.env['FARO_COLLECTOR_URL'] ?? ''));
+
```

Comment:
Using the `dotEnv` package is not that safe since it puts the raw .env file in the build app. To lead with a better example, I think it is better to showcase a usage of the flutter provided json config instead .